src/common.rs

```diff
@@ -474,12 +483,12 @@ impl Info {
         self.frame_control.as_ref()
     }
 
-    /// Returns the bits per pixel
+    /// Returns the number of bits per pixel.
     pub fn bits_per_pixel(&self) -> usize {
         self.color_type.samples() * self.bit_depth as usize
     }
 
-    /// Returns the bytes per pixel
+    /// Returns the number of bytes per pixel.
```

Comment:
This should probably say that the number is rounded up to the nearest byte. Though to be honest I'm not quite sure why we have this function at all. When would you actually want to do rounding when computing pixel size?

Reply:
The bytes per channel are rounded up. For 4bpp all channels are still in individual bytes.